setup.py

```diff
@@ -1,7 +1,7 @@
 import setuptools
 
-with open("README.md", "r") as f:
-    long_description = f.read()
+with open("README.md", "rb") as f:
+    long_description = f.read().decode('utf-8')
```

Comment:
I would add a comment on top of this line, explaining why decoding is important here

Reply:
@florian-fischer-swarm can you do that?

Reply:
I would also use double quotes `"`, so the linter does not complain 🙂 
```suggestion
    long_description = f.read().decode("utf-8")
```